stable/artifactory-ha/values.yaml

```diff
@@ -997,6 +1001,7 @@ artifactory:
       bucketName: artifactory-aws
       path: artifactory/filestore
       endpoint:
+      maxConnections: 50
```

Comment:
@DNewt Thanks for the contribution! Can you please add the same to the `artifactory` chart?

Reply:
@DNewt preferably in same PR :)

Reply:
Yep done :)